index.js

```diff
@@ -110,7 +111,7 @@ const calculateRatio = (color1, color2) => {
     L2 is the relative luminance of the darker of the colors. */
   let lighterLum = Math.max(color1Luminance, color2Luminance);
   let darkerLum = Math.min(color1Luminance, color2Luminance);
-  return ((lighterLum + 0.05) / (darkerLum + 0.05)).toFixed(2);
+  return Number(((lighterLum + 0.05) / (darkerLum + 0.05)).toFixed(2));
```

Comment:
I found a "bug" in `calculateRatio`. The function calculates a ratio but returned a string(`toFixed()` returns a string) which I think don't make much sense. It should return a number.
 I wrapped the whole expression with Number() constructor to ensure the returned value is a number.

Reply:
Makes sense, but this PR is about tests.
So it is out of the scope.